src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+@import './variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  max-width: 100%;
+  min-width: 1024px;
```

Comment:
The `min-width: 1024px;` might cause horizontal scrolling on devices that are narrower than 1024px. It's better to make the layout responsive to different screen sizes.